.changeset/lovely-glasses-cross.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphiql/toolkit': patch
```

Comment:
Let's make this a minor as we're adding new functionality

Reply:
```suggestion
'@graphiql/toolkit': minor
```